src/index.html

```diff
@@ -21,7 +21,449 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+            title="monitor Apple"
+            class="card__img"
+            height="134"
+            width="160"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+            title="monitor Apple"
+            class="card__img"
+            height="134"
+            width="160"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
```

Comment:
You have a lot of duplicated code for each card. Consider creating a reusable component for the card and using a loop to generate each card. This will make your code more maintainable and easier to read. It's a good practice to follow the DRY (Don't Repeat Yourself) principle in coding.